### PR TITLE
CU-862jk516x - Card Component + tailwind.config refactoring

### PR DESCRIPTION
**Pull request by OleksandrMykhailovskyi**
Here are the parts of functional, that have been done in the PR: 

- Configured Card component and its styles. (div and children inside)
- Added hover, focus and focus-within effects.
- Styles for the dark theme.
- Installed a new version of Next that work on Windows.
- Refactored tailwind.config.
- Added the shadows styles for the dark theme.

### CODE REVIEW

**Comment by andrewmarushko** [comment]
Task linked: [CU-862jk516x Card Component](https://app.clickup.com/t/862jk516x)

File: tailwind.config.js [hunk 1]
```diff
@@ -77,6 +78,17 @@ module.exports = {
         tightTitle: 'var(--letter-spacing-title)',
         tightSubtitle: 'var(--letter-spacing-subtitle)'
       },
+      boxShadow: {
+        customSm: 'var(--shadow-small)',
+        customMd: 'var(--shadow-medium)',
```
Comment (andrewmarushko) [inline tailwind.config.js:83]: Сделай более глобальное название кастом не подходит

**Review by andrewmarushko — CHANGES_REQUESTED** [review]

File: tailwind.config.js (hunk 1 above)
Comment (OleksandrMykhailovskyi) [reply tailwind.config.js:83]: а если перебить дефолтные значения у ключей sm и md?

File: styles/globals.css [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 :root {
   /* Primary */
+  --accents-0: #000;
```
Comment (andrewmarushko) [inline styles/globals.css:7]: It's should be default or accent 
Don't use 0

File: styles/globals.css [hunk 3]
```diff
@@ -127,14 +143,17 @@
...
   /* Line height */
-  --lh-74: 74px;
+  --line-height-74: 74px;
 
   /* Letter spacing */
   --letter-spacing-title: -0.04em;
   --letter-spacing-subtitle: -0.03em;
+  /* .dark {
+    --shadow-small: 0 0 0 1px var(--accents-2);
+  } */
```
Comment (andrewmarushko) [inline styles/globals.css:156]: Delete it if not needed

File: tailwind.config.js [hunk 4]
```diff
@@ -23,60 +22,101 @@ module.exports = {
...
-        cyan: 'var(--geist-cyan)',
-        'cyan-dark': 'var(--geist-cyan-dark)',
-        'highlight-purple': 'var(--geist-highlight-purple)',
-        'highlight-magenta': 'var(--geist-highlight-magenta)',
-        'highlight-pink': 'var(--geist-highlight-pink)',
-        'highlight-yellow': 'var(--geist-highlight-yellow)',
-          link: 'var(--geist-link-color)',
+        //accents
+        accent: {
+          DEFAULT: 'var(--accents-0)',
```
Comment (andrewmarushko) [inline tailwind.config.js:27]: So if u use like that just leave accent

File: yarn.lock [hunk 5]
```diff
@@ -61,6 +61,14 @@
     "@floating-ui/dom" "^0.5.3"
     use-isomorphic-layout-effect "^1.1.1"
 
+"@fullhuman/postcss-purgecss@^2.1.2":
```
Comment (andrewmarushko) [inline yarn.lock:64]: Also update lock file after changing next version

File: package.json [hunk 6]
```diff
@@ -33,7 +33,7 @@
     "eslint-config-next": "13.2.4",
     "inter-ui": "^3.19.3",
     "lucide-react": "^0.130.0",
-    "next": "^13.3.0",
+    "next": "13.3.1-canary.4",
```
[the remark this answers is not in the document]
Comment (OleksandrMykhailovskyi) [reply package.json:36]: but we arranged, that we will try this version of next. "next": "^13.3.0 Does not work in Windows :(

**Review by andrewmarushko — APPROVED** [review]

**Pull request merged**